test/imp/test-stepf.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+IGNORE=$($KPROVE --haskell-backend-command "$KORE_REPL -r --repl-script run-stepf-repl-script-spec.k.repl" -d . -m VERIFICATION sum-spec.k)
+
+if grep -q "Config at node 10" shouldBeAt10.out; then
+    if grep -q "Config at node 18" shouldBeAt18.out; then
+        diff finished.out finished.out.golden
```

Comment:
I think not using the `test-stepf.sh.out.golden` file here is confusing, and you'd need to manually regenerate the `finished.out.golden` if you ever change the `Proof completed...` text. Would replacing this line with `cat finished.out` work?